pkg/util/common/common.go

```diff
@@ -112,3 +113,21 @@ func valueToInterface(value reflect.Value) interface{} {
 
 	return nil
 }
+
+// GetSliceOfStringMap is a utility function that returns a config key expected to be a slice of map[string]interface{} as a slice of map[string]string, assuming the underlying types can be cast to string.
```

Comment:
This seems ultra specific to your use case, no ? Should it be part of the `common` package ?

Also the the docstring make it sound that we're simply casting types here where we're converting everything to string actually.